.github/workflows/reusable_basic.yml

```diff
@@ -48,7 +48,7 @@ jobs:
             install_tbb: 'ON'
             disable_hwloc: 'OFF'
             link_hwloc_statically: 'OFF'
-            cmake_ver: 'latest'
+            cmake_ver: '3.29.0'
```

Comment:
can't we use 3.28, as the min version supported of this feature?

Reply:
done